src/applications/gi/tests/actions/index.unit.spec.js

```diff
@@ -0,0 +1,87 @@
+import { expect } from 'chai';
```

Comment:
Added some unit tests for my action since it can dispatch 4 different action types.